frontend/src/utils/utils.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export function isDefined<T>(anyVal: T | undefined | null): anyVal is T {
```

Comment:
Where are you using it?

Reply:
to avoid writing `x != null && x != undefined` whenever I want to check it. 
(don't suggest if (x) because empty strings, lists, zero values will be implicitly converted to false in this case)

what you suggest using instead?

Reply:
oh, yes, nowhere. perhaps worth removing